### ISSUE TITLE
Turrets not working

### ISSUE THREAD

**Issue by PhantomQ** [msummers123/Movecraft-3#169]
Ok, so I've got the latest update, since the last 3 updates my Turret.craft hasn't worked. The plugin won't recognize that 'Turret' is even a craft, let alone get it as a Subcraft. Before those however, it worked. So what changed? We haven't changed anything in it's make up. We don't have any new plugins and none that use signs other than this one. 

Here is the .craft file:

name: Turret
maxSize: 1000
minSize: 10
allowedBlocks: 
    - 8
    - 9
    - 10
    - 11
    - 17
    - 20
    - 23
    - 25
    - 26
    - 29
    - 33
    - "35:8"
    - 41
    - 43
    - 44
    - 47
    - 50
    - 52
    - 53
    - 54
    - 55
    - 57
    - 58
    - 61
    - 62
    - 63
    - 64
    - 65
    - 68
    - 69
    - 70
    - 71
    - 72
    - 77
    - 85
    - 89
    - 92
    - 93
    - 94
    - 95
    - 96
    - 101
    - 102
    - 107
    - 113
    - 114
    - 117
    - 118
    - 123
    - 124
    - 125
    - 126
    - 130
    - 131
    - 133
    - 134
    - 135
    - 117
    - 118
    - 123
    - 124
    - 125
    - 126
    - 130
    - 131
    - 133
    - 134
    - 135
    - 136
    - 138
    - 140
    - 143
    - 144
    - 145
    - 146
    - 147
    - 148
    - 149
    - 150
    - 151
    - 152
    - 154
    - 157
    - 158
    - "159:7"
    - "159:8"
    - "159:15"
    - 165
    - 171
    -173

forbiddenBlocks:
    - 7

canFly: true
canCruise: false
maxHeightLimit: 144
collisionExplosion: 0.0
allowHorizontalMovement: false
allowVerticalMovement: false
speed: 5.0
tryNudge: false
sinkPercent: 99.0
flyblocks:
    35:
        - 0.0
        - 100.0
    49:
        - 0.0
        - 10.0

**Comment by keough99** [comment]
I havnt updated yet but I've had the same trouble b4 look through the craft files incase there's a copy as I found that movecraft likes to make new craft files even when there are already ones that are exactly the same that movecraft created

**Comment by PhantomQ** [comment]
Nope, no doubling up. All craft files are listed with only one of each.

**Comment by keough99** [comment]
Is the type named properly as turret.craft not anything like for example turret.craft.txt

**Comment by PhantomQ** [comment]
Yes it is named correctly. 
Turret.craft
Is the file name. Like I said, before updating it worked fine, then update and now nothing. I did manage to do a temp fix for all my servers turrets by replacing their signs with TorpTurret. Which the only difference between the TorpTurret and Turret crafts are the name. Everything else is the same. Both of these crafts are two of the default crafts that came with the plugin.

**Comment by msummers123** [comment]
The most common cause of this is some kind of typo in the craft file. For example, it can't have any tabs anywhere in it. Unfortunately copying the craft file to this site removes all formatting. Can you link a copy of your server.log file, I need to see your server console's output on startup. It may be saying something about the craft file in there.

**Comment by WanderingBread** [comment]
Its the final item data, -173 requires a space inbetween it and the dash.
